src/Tizen.NUI.Components/Controls/Toast.cs

```diff
@@ -117,6 +150,34 @@ public Toast(string style) : base(style)
             Initialize();
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public TextLabel Textlabel
+        {
+            get
+            {
+                if (null == textLabel)
+                {
+                    textLabel = new TextLabel()
+                    {
+                        PositionUsesPivotPoint = true,
+                        ParentOrigin = Tizen.NUI.ParentOrigin.Center,
+                        PivotPoint = Tizen.NUI.PivotPoint.Center,
+                        WidthResizePolicy = ResizePolicyType.UseNaturalSize,
+                        HeightResizePolicy = ResizePolicyType.UseNaturalSize,
+                        HorizontalAlignment = HorizontalAlignment.Center,
+                        VerticalAlignment = VerticalAlignment.Center,
+                        TextColor = Color.White
+                    };
+                    Add(textLabel);
+                }
+                return textLabel;
+            }
+            set
```

Comment:
Can you make it internal API?
I think we should not allow users to replace sub-components of a components for now. It can cause unexpected behaviors.